pages/docs/components/layout/container.mdx

```diff
@@ -43,14 +39,30 @@ To contain any piece of content, wrap it in the `Container` component.
 
 ## Container Size
 
-To set the size of a Container, use the `maxW` attribute.
+By default, the `Container` component sets the `maxWidth` of the content to 60
+characters (`60ch`) but you can customize this by passing custom `maxWidth`
+values or changing the size tokens defined in
+[theme.sizes](https://chakra-ui.com/docs/styled-system/theming/theme#sizes).
```

Comment:
I'd leave this as before with theme.size.container to leave no room for confusion.

Reply:
Understandable. Easy enough! 😄